CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+
+## Project setup
+
+1. Install root packages `npm install`
+2. Run `npm run bootstrap`
+
+## Testing examples
+
+1. Navigate to specific example project (angular or nextjs) `cd examples/angular`
+2. Open it in new VSCode instance `code .`
```

Comment:
Why does it need to be opened in VSCode?

Reply:
Should we be more generic and write something like "Open it with you default code editor, e.g. `code .`

Reply:
I would omit this step completely, since it has nothing to do with the step that comes after.

Reply:
it does! If you run lint inside monorepo some weird things are happening. You told me that we should test linter rules bu opening them in separate VSCode instance outside monorepo.

Reply:
I think you are assuming that one needs to open VSCode and use its terminal to run commands. I would not mention VSCode at all. These steps are purely about running commands in any terminal.

The issue we talked about it is  about the ESLint plugin for VSCode that doesn't work in monorepos properly.